schematic/manifest/generator.py

```diff
@@ -248,12 +248,15 @@ def _get_column_data_validation_values(
         spreadsheet_id,
         valid_values,
         column_id,
+        strict,
         validation_type="ONE_OF_LIST",
         custom_ui=True,
         input_message="Choose one from dropdown",
-    ):
+    ):  
 
-        strict = CONFIG["style"]["google_manifest"].get("strict_validation", True)
+        # set validation strictness to config file default if indicated.
+        if strict == "default":
+            strict = CONFIG["style"]["google_manifest"].get("strict_validation", True)
```

Comment:
It's strange that `strict` can be either a string (`"default"`) or a boolean (`True`). Is there a way to avoid this? If `"default"` is simply intended to act as a flag value for "nothing was provided", then I suggest using `None` instead since it's more conventional.

Reply:
Alternatively, you could avoid confusion by using different names for the string and boolean variables.

Reply:
Changed `'Default'` to `None`